docs/developers/environment.md

```diff
@@ -36,35 +36,110 @@ sudo yum install curl tar wget clang pkg-config libssl-dev jq build-essential gi
 ```
 <!-- markdownlint-enable MD013 -->
 
+### macOS Dependency Installation
+
+#### 🍺 Installing Homebrew
+
+[Homebrew](https://brew.sh/) is a package manager for macOS and Linux and will
+allow you to install your dependencies.
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Remember to run the commands in the output that are similar to:
+
+```bash
+==> Next steps:
+- Run these three commands in your terminal to add Homebrew to your PATH:
+    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> /Users/joshstein/.zprofile
+    echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/joshstein/.zprofile
+    eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
+#### 🗄 Install wget and jq
+
+Using Homebrew, in your terminal:
+
+```bash
+brew install wget && brew install jq
+```
+
+[wget](https://www.gnu.org/software/wget/) is an internet file retriever and
+[jq](https://stedolan.github.io/jq/) is a lightweight command-line JSON
+processor.
+
 ## Install Golang
 
 Celestia-app and celestia-node are written in [Golang](https://go.dev/) so we
 must install Golang to build and run them.
 
-```sh
-ver="1.19.1"
+> If you are running an Arabica node, you will need to install Golang Version
+> 1.19.1
+
+### If you are running Ubuntu Linux 20.04 (LTS) on an AMD Machine
+
+```bash
+ver="1.18.2"
 cd $HOME
 wget "https://golang.org/dl/go$ver.linux-amd64.tar.gz"
 sudo rm -rf /usr/local/go
 sudo tar -C /usr/local -xzf "go$ver.linux-amd64.tar.gz"
 rm "go$ver.linux-amd64.tar.gz"
 ```
 
+### If you are running Ubuntu Linux 20.04 (LTS) on an ARM Machine or Raspberry Pi
```

Comment:
Can you make all the following code snippets as tabs?

https://docusaurus.io/docs/markdown-features/tabs

It'll be cleaner, happy to help if you need my input